library-best-practices/JLBP-12.md

```diff
@@ -0,0 +1,20 @@
+[JLBP-12] Make level of support and API stability clear
+-------------------------------------------------------
+
+- Clearly describe the lifecycle status (and corresponding level of expected
+  support) of the library. Examples include:
+  - Incubating (no support expected)
+  - Stable (SLOs in place to handle issues)
+  - Maintenance (only bugs will be addressed)
+  - Deprecated / Abandoned / Inactive (no work is expected)
+- Clearly state the API stability clients can expect from the library
```

Comment:
Does the difference between API stability and behavioral stability matter for this JLBP? It seems libraries should document both of these together, for the most part.

Reply:
Yeah I think library owners should be clear about both. I will add that.